cmd/test/main.go

```diff
@@ -70,9 +71,19 @@ func NewTestCommandFlags() *TestCommandFlags {
 		Verbose:              false,
 		PolicyName:           "",
 		SchemaLocations:      make([]string, 0),
+		SkipValidation:       "",
 	}
 }
 
+func validateSkipValidationFlag(flags *TestCommandFlags) error {
+	if flags.SkipValidation != "" && flags.SkipValidation != "schema" {
+		return fmt.Errorf("invalid --skip-validation option - %q\n"+
+			"Valid values are - schema", flags.SkipValidation)
+
+	}
+	return nil
+}
+
```

Comment:
Maybe this will be more robust for the future:
```
supportedSkipValidation := []string{"schema"}
	
if slices.Contains(supportedSkipValidation, flags.SkipValidation) {
return fmt.Errorf("invalid --skip-validation option - %q\n"+
	"Valid values are - schema", flags.SkipValidation)

}
```
When you print the valid values you can use the string array

Reply:
We can extract all the validation to a sort of a util, can open an OSS issue for that